src/components/used-storage.js

```diff
@@ -9,8 +9,8 @@ function UsedStorage({ usedBytes, maxBytes, showLabel }) {
   const { t } = useTranslation('usedStorage');
   const { uiLocale } = useLocale();
 
-  const percent = maxBytes ? usedBytes * 100 / maxBytes : 100;
-  const displayedPercent = `${Math.round(percent)} %`;
+  const percent = maxBytes ? usedBytes * 100 / maxBytes : 101;
+  const displayedPercent = percent > 100 ? '> 100 %' : `${Math.round(percent)} %`;
```

Comment:
Would it be maybe more useful to display the real percentage when > 100% ? Feels a bit hacky this way.
Of course, the bar would just be filled 100%, just the displayed value maybe could be the real one?